docs/resources/acl_token.md

```diff
@@ -1,20 +1,22 @@
 ---
-layout: "consul"
-page_title: "Consul: consul_acl_token"
-sidebar_current: "docs-consul-resource-acl-token"
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "consul_acl_token Resource - terraform-provider-consul"
+subcategory: ""
 description: |-
-  Allows Terraform to create an ACL token
+  The consul_acl_token resource writes an ACL token into Consul.
+  ~> NOTE: The consul_acl_token resource does not save the secret ID of the generated token to the Terraform state to avoid leaking it when it is not needed. If you need to get the secret ID after creating the ACL token you can use the consul_acl_token_secret_id datasource.
 ---
 
-# consul_acl_token
+# consul_acl_token (Resource)
 
 The `consul_acl_token` resource writes an ACL token into Consul.
 
-## Example Usage
+~> **NOTE:** The `consul_acl_token` resource does not save the secret ID of the generated token to the Terraform state to avoid leaking it when it is not needed. If you need to get the secret ID after creating the ACL token you can use the [`consul_acl_token_secret_id`](/docs/providers/consul/r/acl_token.html) datasource.
 
-### Basic usage
+## Example Usage
```

Comment:
```suggestion
## Example usage
```
the docs team is moving towards sentence style capitalization for titles so maybe we could do the same here